locust_plugins/users/socketio.py

```diff
@@ -5,6 +5,7 @@
 import gevent
 import websocket
 from locust import User
+from websocket._abnf import *
```

Comment:
this is never used, is it?

Reply:
I push the wrong commit. Let me fix it rq